tasks/broad/UnmappedBamToAlignedBam.wdl

```diff
@@ -137,16 +137,9 @@ workflow UnmappedBamToAlignedBam {
     }
   }
 
-  # Sum the read group bam sizes to approximate the aggregated bam size
-  call Utils.SumFloats as SumFloats {
```

Comment:
I think this task is mentioned in the [WGS Tasks and Tools table](https://broadinstitute.github.io/warp/docs/Pipelines/Whole_Genome_Germline_Single_Sample_Pipeline/README/#workflow-tasks-and-tools). Would you mind updating that?

Reply:
Mischief managed! 

Thank you!